### DISCUSSION TITLE
[bug]: How to uninstall zinit

### DISCUSSION THREAD

**Discussion by siathalysedI** [zdharma-continuum/zinit#533] [Q&A]
### What happened?

I went through the documentation and could not find how to uninstlal zinit

### Steps to reproduce

1. Install zinit
2. Uninstall zinit

### Relevant output

```shell
Not applicable as I could not uninstall
```

### Operating System & Version

OS: msys | Vendor: pc | Machine: x86_64 | CPU: x86_64 | Processor: unknown | Hardware: x86_64

### Zsh version

5.9

### Terminal emulator

xterm

### If using WSL on Windows, which version of WSL

None

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by vladdoster** [comment]
@intfrr, 

You can run:

```shell
rm -rf "${ZINIT[HOME_DIR]}"
```

Then remove the initialization code in you zsh configuration.

**Reply by rsbkmr** [reply]
how to remove initialization code?

**Reply by sjsdouglas** [reply]
> how to remove initialization code?

You can do this by deleting the lines between
```
### Added by the Zinit installer
...
### End of Zinit installer block
```
at the end of the **_~/.zshrc_** file.
